lib/jsdom/living/nodes/ChildNode-impl.js

```diff
@@ -1,6 +1,38 @@
 "use strict";
+const { createNode } = require("../node");
 
 class ChildNodeImpl {
+  after(...nodes) {
+    const parent = this.parentNode;
+    if (parent) {
+      let viableNextSibling = this.nextSibling;
+      let idx = viableNextSibling ? nodes.indexOf(viableNextSibling) : -1;
+
+      while (idx !== -1) {
+        viableNextSibling = viableNextSibling.nextSibling;
+        if (!viableNextSibling) {
+          break;
+        }
+        idx = nodes.indexOf(viableNextSibling);
+      }
+      parent.insertBefore(createNode(this._ownerDocument, nodes), viableNextSibling);
+    }
+  }
+
+  before(...nodes) {
```

Comment:
While after() seems to follow the spec, before() and replaceWith() do not; they are missing a lot of steps. Either add a comment explaining why, or update to match the spec. See https://dom.spec.whatwg.org/#dom-childnode-before